search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -432,4 +432,63 @@ private IndexDocument crateSampleIndexDocument(String indexName, String jsonFile
 
         return new IndexDocument(eventConsumptionAttributes, jsonNode);
     }
-}
\ No newline at end of file
+
+    @Test
+    void shouldNotReturnTicketsAggregationsWhenNotRequested() throws ApiGatewayException, IOException,
```

Comment:
It seems like most tests in this class create an index and delete it afterwards. This is prone to error as any exception thrown is the test leads to an index that is never deleted. Consider refactoring index creation to be done in methods annotated with @BeforeAll and @AfterAll. Might not be a big issue in this case at the test container is deleted after the tests are run anyway, but it makes the test code more readable with less code.

Reply:
Done.